packages/mobile/src/screens/explore-screen/components/ExploreCarousel.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+import { Flex, Text } from '@audius/harmony-native'
+import { makeStyles } from 'app/styles'
+
+const useStyles = makeStyles(({ palette, spacing, typography }) => ({
+  exploreCarousel: {
+    marginHorizontal: spacing(-4)
+    // paddingLeft: spacing(4)
+  },
+  exploreCarouselContent: {
+    paddingLeft: spacing(4)
+  }
+}))
+
+export const ExploreCarousel = (props) => {
+  //       title={messages.featuredPlaylists}
+  //   data={exploreContent?.featuredPlaylists}
+  //   Card={CollectionCard}
+  const styles = useStyles()
+
+  const { title, list } = props
+  const carouselList = React.cloneElement(list, {
```

Comment:
ah how clever, lets try something else though ultimately, prob best to pair on this